messages/pt.json

```diff
@@ -250,5 +250,8 @@
   "admin/pages.admin.content.general.description.placeholder": "Descrição",
   "admin/pages.admin.content.general.seo.page-title": "<Título da página>",
   "admin/pages.admin.content.general.seo.description": "<A descrição é o texto geralmente utilizado pelo Google para a exibição dos resultados de texto das segundas e terceiras linhas dos resultados de busca, logo abaixo do título>",
-  "admin/pages.admin.content.content.section-title": "Conteúdo"
+  "admin/pages.admin.content.content.section-title": "Conteúdo",
+  "admin/pages.admin.content.not-supported.title": "Funcionalidade não suportada :/",
+  "admin/pages.admin.content.not-supported.description": "O tema instalado na sua loja não suporta a criação de páginas institucionais. Caso necessite criar páginas, escolha algum outro tema que suporte essa funcionalidade ou, caso utilize algum tema personalizado, solicite ao desenvolvedor que siga as instruções indicadas no link abaixo.",
```

Comment:
`páginas institucionais` = `content pages`?

Reply:
`páginas institucionais` makes sense in portuguese, but `institutional page` is a bad translation to it.